stdlib/test/builtin/test_simd.mojo

```diff
@@ -1546,6 +1546,41 @@ def test_contains():
     assert_false(0 in y or 5 in y)
 
 
+def test_comparison():
+    alias I = SIMD[DType.int32, 4]
+    var i = I(-2, 0, 2, 10)
+
+    assert_true(i.__lt__(I(-1, 1, 3, 11)).reduce_and())
+    assert_false(i.__lt__(I(-2, -1, 1, 9)).reduce_or())
+    var mixed_lt = i.__lt__((I(1, 1, 2, 9)))
+    assert_true(mixed_lt[0] and mixed_lt[1])
+    assert_false(mixed_lt[2] or mixed_lt[3])
```

Comment:
I know this is shorter, but just to make the tests failures more specfic (and because we're paranoid about implicit conversions), could you please change these to individual asserts for each element?

Reply:
> I know this is shorter, but just to make the tests failures more specfic (and because we're paranoid about implicit conversions), could you please change these to individual asserts for each element?

addressed in [60afbc7](https://github.com/modularml/mojo/pull/3221/commits/60afbc7ac33a406de178002660a561dcad6f462a)